modules/platforms/dotnet/Apache.Ignite/Internal/Proto/MessagePackReaderExtensions.cs

```diff
@@ -189,8 +170,6 @@ public static bool TryReadNoValue(this ref MessagePackReader reader)
                 throw new IgniteClientException(
                     $"Expected {expectedLength} bytes for {expectedType} extension, but got {hdr.Length}.");
             }
-
-            return (int)hdr.Length;
```

Comment:
@ptupitsyn How it's related to IgniteUUID -> UUID?

Reply:
@sanpwc I've removed `IgniteUuid` methods and the return value of this method is no longer used anywhere.